contracts/reverseRegistrar/L2ReverseRegistrar.sol

```diff
@@ -405,7 +257,7 @@ contract L2ReverseRegistrar is
     function name(
         bytes32 node
     ) external view virtual override returns (string memory) {
```

Comment:
Is it necessary to override these at all? Couldn't we eliminate them and just rename `_name` to `name`?

Reply:
`_name(bytes32)` is used by both `L2ReverseRegistrar.name(bytes32)` and `DefaultReverseRegistrar.name(address)`. I kept `_name(bytes32)` as private because I don't want to expose the function to `DefaultReverseRegistrar.name`

Reply:
Where can I see this implementation of `DefaultReverseRegistrar.name(address)`? It doesn't seem to be in this PR.

Regardless, I don't see how that precludes it calling a parent class's `name` method with a different signature, if it's defined?

Reply:
`Where can I see this implementation of DefaultReverseRegistrar.name(address)? `

It's on the evmgateway PR https://github.com/ensdomains/evmgateway/pull/33/files#diff-f685401f720e672a03e9ec861162971688a4d78c3c084741d6c1edd1f2b32e53R50

I can change to let `DefaultReverseRegistrar` have `name(address)` and `name(node)` if you say it's not too confusing